Sources/SwiftDocC/Servers/FileServer.swift

```diff
@@ -90,7 +90,7 @@ public class FileServer {
                 xlog("Tried to load an invalid path: \(path).\nFalling back to serve index.html.")
             }
             mimeType = "text/html"
-            data = self.data(for: path.appendingPathComponent("/index.html"))
+            data = self.data(for: "/index.html")
```

Comment:
Can you explain why this isn't a change in behavior? What's needless about this?

Reply:
Only `path` component of `url` that `data(for:)` took as argument was used. The rest of the components are ignored by DocC server implementations.

Additionally, `HTTPRequest` unlike `URLRequest` doesn't store full URLs, only paths. The authority and scheme components can be assembled from `scheme` and `authority` properties on `HTTPRequest` , but those aren't used anywhere in the DocC codebase, other than a test in this diff below where we have to construct a new `URLRequest` to pass it to `WKWebView` on Darwin platforms.

Reply:
But if `request.path` is `"/something"` then this used to pass `"/something/index.html"` as an argument but now it only passes "index.html". Isn't that returning data for the wrong path?

Reply:
If `request.path` is `"/something"` then that is handled by the `if` branch above, no this `else` branch. The latter branch didn't use `request.path` previously, that logic is unchanged.